src/shared/codelens/pythonCodeLensProvider.ts

```diff
@@ -34,10 +34,11 @@ import {
     InvokeLambdaFunctionArguments,
     LambdaLocalInvokeParams,
     makeBuildDir,
-    makeInputTemplate,
-    waitForDebugPort
+    makeInputTemplate
 } from './localLambdaRunner'
+import { PythonDebugAdapterHeartbeat } from './pythonDebugAdapterHeartbeat'
 
+const PYTHON_DEBUG_ADAPTER_RETRY_DELAY_MILLIS = 1000
```

Comment:
nit: usually we end milisecond stuff with `_MS`

Reply:
Old habit of mine (changed)